planner/optimize.go

```diff
@@ -371,6 +356,56 @@ func optimize(ctx context.Context, sctx sessionctx.Context, node ast.Node, is in
 	return finalPlan, names, cost, err
 }
 
+// OptimizeExecStmt to handle the "execute" statement
+func OptimizeExecStmt(ctx context.Context, sctx sessionctx.Context,
```

Comment:
Can we remove this part in `optimize()`?
<img width="582" alt="image" src="https://user-images.githubusercontent.com/7499936/183033589-35634b30-e567-4a98-a741-d6084032afcb.png">